web-components/src/components/tabs/Tabs.ts

```diff
@@ -1210,34 +1259,59 @@ export namespace Tabs {
                 : {}
             )}"
           >
-            ${repeat(
-              this.tabsFilteredAsHiddenList,
-              () => generateSimpleUniqueId("tabs"),
-              (tab) => html`
-                <md-tab
-                  slot="draggable-item"
-                  .disabled="${tab.disabled}"
-                  .selected="${tab.selected}"
-                  name="${tab.name}"
-                  id="${this.getCopyTabId(tab)}"
-                  aria-label=${tab.ariaLabel}
-                  aria-controls="${this.getAriaControlId(tab)}"
-                  @click="${() => this.handleOverlayClose()}"
-                  tabIndex="${this.tabHiddenIdPositiveTabIndex === tab.id ? 0 : -1}"
-                  ariaRole="menuitem"
-                  .newMomentum=${this.newMomentum}
-                  type=${this.type}
-                  .onlyIcon="${tab.onlyIcon}"
-                >
-                  ${unsafeHTML(tab.innerHTML)}
-                </md-tab>
-              `
-            )}
+            ${this.moreMenuListTemplate}
           </div>
         </md-menu-overlay>
       `;
     }
 
+    private get renderTabSlot() {
+      return html`<slot
+        name="tab"
+        class="${classMap({
+          "visible-tabs-slot": this.direction === "horizontal"
+        })}"
+      ></slot> `;
+    }
+
+    private get renderTabsWithMoreMenu() {
+      return html`
+        <div
+          id="visible-tabs-list"
+          class="visible-tabs-container ${classMap({
+            "md-tab__justified": this.justified && !this.isMoreTabMenuVisible,
+            "md-tab__hug": this.hugTabs,
+            "visible-new-tabs": this.newMomentum
+          })}"
+        >
+          ${repeat(
+            this.tabsFilteredAsVisibleList,
+            () => generateSimpleUniqueId("tabs"),
+            (tab) => html`
+              <md-tab
+                .closable="${tab.closable}"
+                .disabled="${tab.disabled}"
+                .selected="${tab.selected}"
+                name="${tab.name}"
+                id="${this.getCopyTabId(tab)}"
+                aria-label=${tab.ariaLabel}
+                aria-controls="${this.getAriaControlId(tab)}"
+                .isCrossVisible=${true}
```

Comment:
There is still an issue with this approach in that the copy resets the aria controls but they are now in separate shadow doms